Python/Tests/Utilities.Python/EventTaskSources.cs

```diff
@@ -14,6 +14,8 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
+using Microsoft.PythonTools.Analysis.Infrastructure;
```

Comment:
Seems like an unnecessary change?

Reply:
Actually this may need to remain a copy so we don't reference out-of-proc stuff